src/Math-Tests-Matrix/PMMatrixTest.class.st

```diff
@@ -197,6 +197,20 @@ PMMatrixTest >> testFlattenRows [
 	self assert: m flattenRows equals: #(1 2 3 4 5 6 7 8 9) asPMVector
 ]
 
+{ #category : #tests }
+PMMatrixTest >> testIdentityMatrix [
+	|expectedIdentityMatrix identityMatrix|
+
+	expectedIdentityMatrix := PMMatrix zerosRows: 3 cols: 3.
+
+	expectedIdentityMatrix at: 1 at: 1 put: 1.
+	expectedIdentityMatrix at: 2 at: 2 put: 1.
+	expectedIdentityMatrix at: 3 at: 3 put: 1.
+
+	identityMatrix := PMMatrix identity: 3.
+	self assert: identityMatrix equals: identityMatrix.
```

Comment:
You mean `equals: expectedIdentityMatrix.`

As, given the factory method is well named, you can, as an option, inline the local variable. Happy either way. I'll leave exercising the option in your good hands.

Reply:
Fix this before merging

Reply:
My bad, I have fixed it now.